tox.ini

```diff
@@ -58,6 +64,8 @@ basepython = python3.11
 deps =
     wagtail>=5.2
 
+    .[testing]
```

Comment:
I've avoided the extra deps here during the last round of changes as the interactive bit don't need any of the testing requirements per se.

Saying that, we do use dj-database-url which is a testing requirement, so ok

Reply:
Yeah, it is dj-database-url. Could specify it explicitly if that's better?